### ISSUE TITLE
Accessibility: Input Modalities - Label in Name

### ISSUE THREAD

**Issue by AntonLV** [unacms/UNA#5579]
When using hidden texts such as ARIA labels to provide more specific descriptions for links, the visible text must be included in the ARIA label. Visible text and ARIA label should be consistent to ensure that screen reader users do not perceive conflicting information.
For example: A link with the visible text “Learn more” should have an ARIA label such as “Learn more about accessibility”, where the visible text “Learn more” remains identical within the ARIA label.

Every focusable control element must, in the accessibility tree, read out exactly the text that users can also see as a label. The accessible name (derived from <label>, aria-label, aria-labelledby, alt, etc.) must include the visible label.

Related ticket(s): #5046

**Comment by AlexTr** [comment]
Some inputs became hidden, so aria-label input becomes invisible and non-focusable:

<img width="659" height="133" alt="Image" src="https://github.com/user-attachments/assets/9c95c28c-f1cb-4a5f-a5c9-4afdffce1452" />

Check all inputs types to make sure area-label is set for active visible field.
